src/crepe/system/RenderSystem.cpp

```diff
@@ -21,18 +22,36 @@ void RenderSystem::update_camera() {
 
 	for (Camera & cam : cameras) {
 		SDLContext::get_instance().camera(cam);
```

Comment:
Sla `SDLContext` op als een reference member van RenderSystem, dit maakt het later makkelijker met refactoren als `SDLContext` niet langer een singleton is:

RenderSystem.h (in `class RenderSystem`)
```cpp
private:
	SDLContext & context = SDLContext::get_instance();
```